src/main/java/org/mule/extension/validation/api/ValidationExtension.java

```diff
@@ -49,9 +50,8 @@
 @Extensible(alias = "validator-message-processor")
 @Export(
     resources = {"/META-INF/org/mule/runtime/core/i18n/validation-messages.properties"})
-@ErrorTypes(ValidationErrorTypes.class)
-@OnException(ValidationExceptionEnricher.class)
-@Throws(ValidationErrorTypeProvider.class)
+@ErrorTypes(ValidationErrorType.class)
+@Throws(BasicValidationErrorType.class)
```

Comment:
The `Throws` javadoc says it can be used at the method, operation class and extension level, with the most specific being used, hence the class change. I could probably live without it but I thought it would guaranteed anything I missed is `VALIDATION:VALIDATION`.